content/en/docs/how-tos/ai-job-scheduling-kueue-integration.md

```diff
@@ -0,0 +1,771 @@
+
+> **Important**: Make sure you have switched to the context of your hub cluster before proceeding with the installation.
+
+```bash
+# Switch to your hub cluster context
+kubectl config use-context <hub-cluster-context>
+
+# Set the Kueue version you want to install
+VERSION=v0.9.0  # Replace with the latest version, or working version you like
+
+# Install required Kueue CRDs on hub cluster
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/kueue/$VERSION/config/components/crd/bases/kueue.x-k8s.io_clusterqueues.yaml
+
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/kueue/$VERSION/config/components/crd/bases/kueue.x-k8s.io_localqueues.yaml
+
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/kueue/$VERSION/config/components/crd/bases/kueue.x-k8s.io_resourceflavors.yaml
+
+# Verify CRDs are installed
+kubectl get crds | grep kueue
+```
+
+You should see output similar to:
+```
+clusterqueues.kueue.x-k8s.io                       2025-10-17T17:50:30Z
+localqueues.kueue.x-k8s.io                         2025-10-17T17:50:33Z
+resourceflavors.kueue.x-k8s.io                     2025-10-17T17:50:35Z
+```
+
+## Step-by-Step Guide
+
+### 1. Create Resource Flavor on Hub
+
+First, create a ResourceFlavor that defines the available resource types:
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1 
+kind: ResourceFlavor  
+metadata:  
+  name: default-flavor
+spec:
+  # This ResourceFlavor will be used for all the resources
+```
+
+A ResourceFlavor is an object that represents the variations in the nodes available in your cluster by associating them with node labels and taints. 
+
+
+### 2. Configure ClusterQueue on Hub
+
+Set up a ClusterQueue for the workloads:
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: cluster-queue
+spec:
+  cohort: ai-training
+  namespaceSelector: {} # Available to all namespaces  
+  queueingStrategy: BestEffortFIFO # Default queueing strategy 
+  resourceGroups:
+  - coveredResources: ["cpu", "memory"]
+    flavors:
+    - name: default-flavor
+      resources:
+      - name: cpu
+        nominalQuota: "32"
+      - name: memory
+        nominalQuota: "128Gi"
+```
+> NOTE: The resources need to be listed in the same order as the coveredResources list.
+
+A ClusterQueue is a cluster-scoped object that oversees a pool of resources, including CPU, memory, and GPU. It manages ResourceFlavors, sets usage limits, and controls the order in which workloads are admitted. This will handle the ResourceFlavor that was previously created.
+
+
+### 3. Create Training Namespace on Hub
+
+Create the namespace where your AI jobs will be scheduled:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: training
```

Comment:
multi-cluster training is not easy as scheduling different jobs to many clusters so it's better to find a more neutral name